src/selectors/characterSelectors.js

```diff
@@ -1 +1,4 @@
 export const characters = (state) => state.characters
+
+export const getCharacterById = (id) => (state) =>
```

Comment:
You can just call this selector function `character` since you are selecting a character